buildlib/pr/tests.yml

```diff
@@ -30,6 +30,9 @@ jobs:
       - bash: |
           source ./buildlib/az-helpers.sh
           az_init_modules
+          if [ "${{ parameters.name }}" == "gpu" ]; then
```

Comment:
minor:
can have another wrapper which would accept the name, so that this `if` will be hidden there (you use this `if` twice already)

Reply:
Done, thanks!